dataactvalidator/interfaces/stagingTable.py

```diff
@@ -83,7 +83,7 @@ def createTable(self, fileType, filename, jobId, tableName=None):
             else:
                 classFieldDict[newKey] = Column(fieldTypeName)
             # First record will hold field names
-            fieldNameMap[newKey] = key
+            fieldNameMap[str(newKey)] = str(key)
```

Comment:
In Python 2.7, I think this would break if there actually was a non-ascii character in `newKey` or `key`. That said, this is a pattern we have all over the code base, so I vote to leave it as a further incentive to update to Python 3.